server/profile.go

```diff
@@ -15,12 +15,80 @@
 package main
 
 import (
-    "net/http" 
+    "net/http"
+    "cloud.google.com/go/datastore"
+    "github.com/gorilla/mux"
+    "context"
+    "fmt"
+    "google.golang.org/api/iterator"
+    "log"
+    "encoding/json"
 )
 
+func CheckDatastore(w http.ResponseWriter, r *http.Request) {
+    ctx := context.Background()
+    client, err := datastore.NewClient(ctx, "lauraod-step-2020")
+    if err != nil {
+        w.WriteHeader(http.StatusOK)
+        w.Write([]byte(`first save error`))
+    }
+    query := datastore.NewQuery("User")
+    it := client.Run(ctx, query)
+    for {
+        var user User
+        _, err := it.Next(&user)
+        if err == iterator.Done {
+            break
+        }
+        if err != nil {
+            log.Fatalf("Error fetching next task: %v", err)
+        }
+        fmt.Fprintf(w, "User %q, ID %q\n", user.Name, user.ID) 
+    }
+    w.Write([]byte(`done`))
+} 
+
+
+
 func GetProfile(w http.ResponseWriter, r *http.Request) {
-	w.WriteHeader(http.StatusOK)
-    w.Write([]byte(`called profile`))
+    ctx := context.Background()
+    dsClient, err := datastore.NewClient(ctx, "lauraod-step-2020")
```

Comment:
Let's move this string into a config file (and possibly put the config into gitignore)

Reply:
Oops, was supposed to do that, will do it now